src/ResourceManager/AzureBatch/Commands.Batch/Az.Batch.psd1

```diff
@@ -150,7 +150,7 @@ PrivateData = @{
         ReleaseNotes = 'Initial Release with PowerShell and PowerShell Core Support'
 
         # Prerelease string of this module
-        # Prerelease = ''
+        Prerelease = 'preview'
```

Comment:
@matthchr you can revert the change here -- for `Az`, we're following the semantic versioning pattern that `0.x.x` is preview (or unstable), so we won't need to do `0.x.x-preview`